lib/presentation/organisms/intro/tips_organism.dart

```diff
@@ -29,49 +51,58 @@ class TipsOrganism extends StatelessWidget {
         )
         .toList();
 
-    return MarginedExpandedAtom(
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          TopBarMolecule(
-            title: 'efficient_tips'.tr(args: [studyType.tr()]),
-            topBarType: TopBarType.none,
-            margin: false,
-            translate: false,
-          ),
-          const SeparatorAtom(variant: SeparatorVariant.farApart),
-          Center(
-            child: Platform.isAndroid
-                ? ButtonAtom(
-                    variant: ButtonVariant.primary,
-                    onPressed: () => DndController.instance.toggleDnd(),
-                    text: 'dnd',
-                  )
-                : const TextAtom('dnd_ios'),
-          ),
-          ListView.builder(
-            shrinkWrap: true,
-            itemBuilder: (BuildContext context, int index) {
-              final Tip tip = beforeStudyTips[index];
+    return dndEnable == null
+        ? Center(child: CircularProgressIndicator())
```

Comment:
Please create atom for CircularProgressIndicator

Reply:
Also let's change it to a nicer version
if(dndEnable == null){
    return Center(child: CircularProgressIndicator());
}

return MarginedExpandedAtom(...